### PR TITLE
Add docs for superuser flow

### PR DESCRIPTION
**Pull request by yiannistri**
**What changed?**
Added docs for superuser login flow.

**Why?**
To assist end users.

**How did you test it?**
N/A

**Release notes**
N/A

**Documentation Changes**
N/A

### CODE REVIEW

File: website/docs/gitops-dashboard.md [hunk 1]
```diff
@@ -33,22 +33,32 @@ Applications being managed by Weave GitOps are displayed in a list. Clicking the
...
 - A list of the 10 most recent commits to the source git repository helping you to verify which change has been applied to the cluster. This includes a hyperlink back to your git provider for each commit.
 
-## Securing the dashboard using OIDC and Kubernetes RBAC
+## Dashboard Login
 
-:::note WORK IN PROGRESS 
-The following instructions describe a feature that is being actively developed and is therefore behind a feature toggle. To enable this feature, set the following OS environment variable:
+There are 2 supported methods for logging in to the dashboard:
+- Login via an OIDC provider
+- Login via the superuser account
```
Comment (foot) [inline website/docs/gitops-dashboard.md:40]: Does superuser have any RBAC rolebindings by default actually?

**Review by foot — APPROVED** [review]
Good stuff! ✨

File: website/docs/gitops-dashboard.md (hunk 1 above)
Comment (yiannistri) [reply website/docs/gitops-dashboard.md:40]: Good question. Do we need to add some? @foot

Comment (foot) [reply website/docs/gitops-dashboard.md:40]: Yeah we should probably add a role-binding onto the role the wego-app SA uses?

Comment (yiannistri) [reply website/docs/gitops-dashboard.md:40]: This should be addressed in https://github.com/weaveworks/weave-gitops/pull/1524

**Pull request merged**